### ISSUE TITLE
[Feature]:Is there anyway to scan a struct or slice to the result of seach or query?

### ISSUE THREAD

**Issue by nisainan** [milvus-io/milvus-sdk-go#800]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

It's hard to use the result of search or query, maybe can provide a method to map the results to a variable, like 'scan' or something else

**Comment by congqixia** [comment]
good point. I shall add some helper methods to iterate or scan result struct
